src/index.js

```diff
@@ -0,0 +1,33 @@
+const Routes = require('./routes')('default')
+
+const Rest = require('./rest')
+const Realm = require('./structures/Realm')
+
+const RealmsBedrockAPI = require('./bedrock/api')
+const RealmsJavaAPI = require('./java/api')
+
+const { Authflow: PrismarineAuth } = require('prismarine-auth')
+
+const PlatformTypes = ['java', 'bedrock']
+
+class RealmAPI {
+  constructor (Authflow = new PrismarineAuth(undefined, undefined, { authTitle: false }), platform) {
```

Comment:
There should only be one way to do something unless there's good justification otherwise. Remove the dependency on prismarine auth here, make it pass an instance here.

Reply:
Sounds good !